### PR TITLE
move to k8s.io v1.20.x and klog/v2 v2.4.0

### PR DESCRIPTION
**Pull request by mythi**
Closes: #505

### CODE REVIEW

**Comment by mythi** [comment]
@rojkov @bart0sh  CI seems OK but I want to run some manual checks still. Any immediate review comments?

**Comment by bart0sh** [comment]
@mythi no comments. /lgtm

**Review by bart0sh — APPROVED** [review]

File: test/e2e/utils/utils.go [hunk 1]
```diff
@@ -51,9 +50,6 @@ func WaitForNodesWithResource(c clientset.Interface, res v1.ResourceName, timeou
 			for t := time.Now(); time.Since(t) < nodeListTimeout; time.Sleep(poll) {
 				nodelist, err := c.CoreV1().Nodes().List(context.TODO(), metav1.ListOptions{})
 				if err != nil {
-					if testutils.IsRetryableAPIError(err) {
-						continue
-					}
```
Comment (rojkov) [inline test/e2e/utils/utils.go:56]: I don't remember how we ended up with this function. Is it a copypaste from k8s/test/e2e?

Comment (mythi) [reply test/e2e/utils/utils.go:56]: I think so yes. `IsRetryableAPIError()` was removed and the commit changed places like this too.

**Comment by mythi** [comment]
I tested QAT/fpgaadmissionwebhook with the e2e tests and those work OK.

**Review by rojkov — APPROVED** [review]

File: test/e2e/utils/utils.go (hunk 1 above)
Comment (mythi) [reply test/e2e/utils/utils.go:56]: https://github.com/kubernetes/kubernetes/commit/64c099d67069f85f2b9b78d7d12fd1a92fcc3d75

**Pull request merged**